ivadomed/training.py

```diff
@@ -123,7 +145,8 @@ def train(model_params, dataset_train, dataset_val, training_params, log_directo
     begin_time = time.time()
 
     # EPOCH LOOP
-    for epoch in tqdm(range(1, num_epochs + 1), desc="Training"):
+    for epoch in tqdm(range(num_epochs), desc="Training", initial=start_epoch):
```

Comment:
@andreanne-lemay : Don't we need a `+1` here?

Reply:
Nope :), we have the right number of epochs when using only one param in range